<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="content_btn">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>833</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Content</string>
   </property>
  </widget>
  <widget class="QPushButton" name="style_btn">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>830</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Style</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel2">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mix_btn">
   <property name="geometry">
    <rect>
     <x>1026</x>
     <y>830</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>섞기</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>880</y>
     <width>75</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>876</x>
     <y>830</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>직접 업로드</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>660</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>그림</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>650</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>스타일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>660</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>융합프로젝트</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
